resume.md

```diff
@@ -32,26 +32,23 @@ Python, Git, Linux, PHP, Javascript, relational databases (MySQL, PostgreSQL), a
 
 # Experience
 
-\cvevent{2019 - July 2023}{Developer}{Technology Advisors}{
-  {Coded custom features requested by clients using best practices defined by CRM vendors.},
-  {Debugging code when custom features no longer function as expected after CRM upgrades.},
-  {Wrote MySQL scripts for "advanced" reports requested by clients which cannot be built through the CRM's UI.},
-  {Maintained CRM developer certifications when required (SugarCRM, Creatio CRM).},
+\cvevent{2019 - July 2023}{Software Engineer}{Technology Advisors}{
```

Comment:
I'm a little hesitant labeling myself an engineer (for now). In my mind software engineers build from scratch or close to it. I was modifying or adding-to vendor made applications which had constraints on what is and what is not allowed. Support and Developer were my job titles. I'm not completely against 'Engineer'.  It certainly sounds better. I suppose I'm a little anxious about over-selling.

Reply:
Don't be. It literally means nothing. It bothers me that we use the title "engineer", which normally means a rigid certification process and accountability for if you e.g. design a bridge all bad and people die. People call themselves software engineers the moment they get their first job writing code, with no degree. You have a degree and years of experience writing code. You're good!